main.js

```diff
@@ -0,0 +1,109 @@
+var products = [];
+var seasonalSales = [];
+const discountSelect = document.getElementById("discountSelect");
+const productDisplayDiv = document.getElementById("productDisplayDiv");
+productDisplayDiv.innerHTML += `<div class="col-md-4"><div class="panel panel-primary"><div class="panel-heading"><h2 class="panel-title">Apparel</h2></div><div class="panel-body" id="apparel"></div></div></div>`;
+productDisplayDiv.innerHTML += `<div class="col-md-4"><div class="panel panel-primary"><div class="panel-heading"><h2 class="panel-title">Furniture</h2></div><div class="panel-body" id="furniture"></div></div></div>`;
+productDisplayDiv.innerHTML += `<div class="col-md-4"><div class="panel panel-primary"><div class="panel-heading"><h2 class="panel-title">Household</h2></div><div class="panel-body" id="household"></div></div></div>`;
+const apparelDiv = document.getElementById("apparel");
+const furnitureDiv = document.getElementById("furniture");
+const householdDiv = document.getElementById("household");
+var discountApplied = { //flags to determine whether discounts have already been triggered or not
```

Comment:
Discounts never get reset. I followed the instructions quite literally:
"As soon as you select one of the seasons, all prices on the page should immediately be discounted by the corresponding percentage. For example, when Spring is chosen, all products in the corresponding Household category should have their prices updated with a 15% discount off the base price."
Obviously that wouldn't be the case IRL. But I haven't had time to implement features not included explicitly in the instructions.